chunky/src/java/se/llbit/chunky/renderer/scene/Sun.java

```diff
@@ -394,6 +497,10 @@ public Vector3 getColor() {
     return new Vector3(color);
   }
 
+  public Vector3 getApparentColor() {
+    return new Vector3(apparentColor);
```

Comment:
Instead of returning copies every time, could we let the consumer copy this vector as needed?

Reply:
Same for the existing `getColor`, we might be able to improve that.